python/mlc_chat/compiler/model/llama_quantization.py

```diff
@@ -0,0 +1,101 @@
+"""
+Quantization specs for Llama2 architecture.
+TODO: add docstring
```

Comment:
From my POV, the docstring looks pretty reasonable. Any specific ones in your mind?